core/server/master/src/main/java/alluxio/master/FaultTolerantAlluxioMasterProcess.java

```diff
@@ -239,4 +244,46 @@ public boolean waitForReady(int timeoutMs) {
       return false;
     }
   }
+
+  @Override
+  protected void startCommonServices() {
+    boolean standbyMetricsSinkEnabled =
```

Comment:
We need all the comments here just because starting the metrics/web-server is not idempotent. So we cannot use a declarative approach by justing comparing the expected state vs the current.

Could you leave a TODO here? Feel free to assign to me like 
`TODO(jiacheng): make starting metrics sink/web server idempotent to simplify the logic`

Thanks!

Reply:
@maobaolong pls leave a todo to me, thx!